test/builder/builder_test.cc

```diff
@@ -142,13 +143,13 @@ class RoadGeometryBuilderTest : public ::testing::Test {
   const maliput::api::LaneId kLaneCId{"lane_c"};
   const maliput::api::LaneId kLaneDId{"unset_id"};
   const LineString3d kLeftLineStringA{Vector3{0., 0., 0.}, Vector3{10., 0., 0.}};
-  const LineString3d kRigthLineStringA{Vector3{0., 5., 0.}, Vector3{10., 5., 0.}};
+  const LineString3d kRightLineStringA{Vector3{0., 5., 0.}, Vector3{10., 5., 0.}};
   const LineString3d kLeftLineStringB{Vector3{0., 5., 0.}, Vector3{10., 5., 0.}};
-  const LineString3d kRigthLineStringB{Vector3{0., 10., 0.}, Vector3{10., 10., 0.}};
+  const LineString3d kRightLineStringB{Vector3{0., 10., 0.}, Vector3{10., 10., 0.}};
   const LineString3d kLeftLineStringC{Vector3{0., 0., 0.}, Vector3{0., 10., 0.}};
-  const LineString3d kRigthLineStringC{Vector3{5., 0., 0.}, Vector3{5., 10., 0.}};
+  const LineString3d kRightLineStringC{Vector3{5., 0., 0.}, Vector3{5., 10., 0.}};
   const LineString3d kLeftLineStringD{Vector3{20., 0., 0.}, Vector3{30., 5., 0.}};
-  const LineString3d kRigthLineStringD{Vector3{20., 0., 0.}, Vector3{30., 5., 0.}};
+  const LineString3d kRightLineStringD{Vector3{20., 0., 0.}, Vector3{30., 5., 0.}};
```

Comment:
Any idea why is it showing changes on the names of the RightLineStrings even though they are the same? This is happening here and on the other parts of the tests.

Reply:
They are not the same haha. `Right` vs `Rigth`

Reply:
I'm feeling my Dyslexia is growing due to this kinda of stuff haha